Druid/DRInterpreterToCompiler.class.st

```diff
@@ -48,7 +48,7 @@ DRInterpreterToCompiler class >> generateDruidJITModel [
 	| primitives bytecodes |
 	primitives := #( #primitiveAdd #primitiveSubtract #primitiveLessThan
 	                 #primitiveGreaterThan #primitiveLessOrEqual
-	                 #primitiveGreaterOrEqual #primitiveEqual #primitiveNotEqual
+	                 #primitiveGreaterOrEqual #primitiveEqual "#primitiveNotEqual"
```

Comment:
😢

Reply:
Yes, we should check why notEqual is broken.